cmd/helm/create.go

```diff
@@ -75,6 +75,9 @@ func newCreateCmd(out io.Writer) *cobra.Command {
 			if len(args) == 0 {
 				return errors.New("the name of the new chart is required")
 			}
+			if len(args) > 1 {
+				return errors.New("command 'create' doesn't support multiple helm-chart creation at a time")
```

Comment:
```suggestion
				return errors.New("command 'create' doesn't support multiple arguments")
```

Minor nitpick on wording!